java/mapreduce/src/java/org/apache/orc/mapred/OrcInputFormat.java

```diff
@@ -58,7 +58,7 @@
    */
   public static boolean[] parseInclude(TypeDescription schema,
                                        String columnsStr) {
-    if (columnsStr == null ||
+    if (StringUtils.isBlank(columnsStr) ||
```

Comment:
BTW, is this enough for ORC column selection to return all null fields?
It looks to me that it returns all value fields. I mean without column selection.